microsoft/testsuites/nvme/nvme.py

```diff
@@ -77,3 +108,230 @@ def nvme_basic_validation(self, environment: Environment, node: Node) -> None:
             assert_that(nvme_namespace).described_as(
                 "nvme devices count should be equal to [vCPU/8]."
             ).is_length(expected_count)
+
+    @TestCaseMetadata(
+        description="""
+        This test case will do following things for each NVMe device.
+        1. Check the the number of errors from nvme-cli before operations.
+        2. Create a partition, filesystem and mount it.
+        3. Create a txt file on the partition, content is 'TestContent'.
+        4. Create a file 'data' on the partition, get the md5sum value.
+        5. Umount and remount the partition.
+        6. Get the txt file content, compare the value.
+        7. Check the the number of errors from nvme-cli after operations.
+        """,
+        priority=1,
+        requirement=simple_requirement(
+            supported_features=[Nvme],
+        ),
+    )
+    def nvme_function_validation(self, environment: Environment, node: Node) -> None:
+        nvme = node.features[Nvme]
+        nvme_namespaces = nvme.get_namespaces()
+        nvme_cli = node.tools[Nvmecli]
+        cat = node.tools[Cat]
+        for namespace in nvme_namespaces:
+            # 1. Check the the number of errors from nvme-cli before operations.
+            error_count_before_operations = nvme_cli.get_error_count(namespace)
```

Comment:
Rather than error count, is it possible to retrieve last error?

Reply:
there are many entries, we need sum up them together.